index.bs

```diff
@@ -6815,7 +6825,7 @@ may return any value, which will be discarded.
         1.  Round |x| to the nearest integer, choosing the even integer if it lies halfway between two,
             and choosing +0 rather than −0.
         1.  Return |x|.
-    1.  If |x| is <emu-val>NaN</emu-val>, +0, −0, +∞, or −∞,
+    1.  If |x| is <emu-val>NaN</emu-val>, +∞, or −∞,
```

Comment:
I'm sorry, I don't understand. This seems to change behavior in a probably-undesirable way, as it now executes the next few steps if x is +0, whereas before this patch those steps are not executed.